### DISCUSSION TITLE
How to fix a 3D model to the ground?

### DISCUSSION THREAD

**Discussion by Abad26Pani** [google/model-viewer#4910] [Q&A]
Hello! I'm doing a university project on augmented reality. 
I want my 3D model to remain fixed on the ground and the user can´t move or scale it.
i don´t know how to to that. :(

**Comment by elalish** [comment]
You can use `ar-scale="fixed"` to fix the scale. We don't allow fixing the position, as it's too likely to be wrong - estimates of where the floor is improve over time, and it's difficult for a user to place the object in the correct position without the ability to reposition it.
